.github/workflows/publish.yml

```diff
@@ -87,36 +80,29 @@ jobs:
         build_target:
           - platform: linux/amd64
             built_on: ubuntu-22.04
-            meta_json: meta.json
           - platform: linux/arm64
             built_on: ubuntu-22.04-arm
-            meta_json: meta.json
           - platform: darwin/arm64
             built_on: macos-14
-            meta_json: meta.json
           - platform: darwin/amd64
             built_on: macos-13
-            meta_json: meta.json
           - platform: windows/amd64
             built_on: windows-2019
-            meta_json: meta.windows.json
     runs-on: ubuntu-latest
     steps:
       - name: Install Viam CLI
         run: |
           sudo curl -o /usr/local/bin/viam https://storage.googleapis.com/packages.viam.com/apps/viam-cli/viam-cli-stable-linux-amd64
           sudo chmod a+rx /usr/local/bin/viam
       # Although we don't need to check out any of the code, we do need to check out meta.json.
-      - name: Download the build artifacts
+      - name: Checkout
```

Comment:
nit: comment above is stale.

Reply:
Not stale: we still need to check out meta.json or else the Viam CLI won't know how to upload the module, and we do check it out in the very next step. TBH, it used to be more stale than it is now (we used to download it from Github artifacts instead of checking out the repo).